technology-considerations/internet_explorer.md

```diff
@@ -0,0 +1,40 @@
+# Developing and Testing for Internet Explorer
+
+## Problem
+
+A key deliverable for the TDR project is to support users of Internet Explorer.
+ 
+This raises some operational challenges for development and testing:
+* Development is undertaken on Linux machines which do not support Internet Explorer
+* Testing of code on Internet Explorer
+ 
+## Possible Approaches
+ 
+### Additional Hardware for Supporting Internet Explorer
+
+Provide developer's with machines which support Internet Explorer and allow them to develop on
```

Comment:
Typo: developer's should be developers.

Same on line 37.

Reply:
Fixed